database/migrations/2017_07_30_205523_create_reciters_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRecitersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reciters', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
```

Comment:
We should also add a `slug` column for reciters. So that we can make URLs like `/nadeem-sarwar/`